ansible_roles/wordpress/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: enable epel
+  yum:
+    name: epel-release
+
+- name: Install Wordpress
+  yum:
+    name: wordpress 
+
+- name: Fetch random salts for WordPress config
+  local_action: command curl https://api.wordpress.org/secret-key/1.1/salt/
+  register: "wp_salt"
+  become: no
+  become_method: sudo
+
+- name: Copy WordPress config file
+  template: 
+    src: wp-config.php
+    dest: /etc/wordpress/
+
+- name: Copy conf for apache
+  template:
+    src: wordpress.tpl
```

Comment:
the convetion is to use j2 extension not tpl to make it clear this are jina2 templates

Reply:
also you seem to use that template as is without any vars inside it's faster to use copy in this case.